### DISCUSSION TITLE
Off topic / but related... would Next.js / Vercel offers anything for coding on the cloud?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#42698] [Help] [answered]
So... is there a way to code while you are seating away from your base computer?

I use VS code in my main computer.  Is there a way to pull the same code else where and work on it while I am on the road? 

I am already using Vercel to host the Next.js codes.... does Vercel offer a cloud environment for coding too?

**Comment by icyJoseph** [comment, marked as answer]
I remember Vercel was showing a demo for a product called Next.js live: https://nextjs.org/blog/next-11#nextjs-live-preview-release not sure where that's at.

I would say with Stackblitz, https://stackblitz.com/, you could login with your GitHub account, and open a project there, push changes and if that repo is also connected to Vercel, then you should be able to see the preview deploys there as well. You'll also be able to see project changes as you develop, because it has a built-in runtime.

Also on Github repositories, if you change the `.com` for `.dev` it'll open a text editor - or if you press `.`, based of VS code directly on the browser, but you can't really run the project, I think. 

And there's also Codespaces, from GitHub too, but I don't have access to it, so I can't vouch for it just yet.

![Screenshot 2022-11-09 at 22 19 53](https://user-images.githubusercontent.com/21013447/200944077-0be9fadf-d635-44e7-be06-03961bf18e0d.png)

One last thing, when opening projects on third parties, is good to setup 2FA in all accounts that allow for it.

**Reply by Elindo586** [reply]
Thanks for the feedback... this is good information...
